static/js/drafts.js

```diff
@@ -101,6 +101,30 @@ export const draft_model = (function () {
         save(drafts);
     };
 
+    exports.getDraftsIdByStreamAndTopic = function (stream, topic) {
+        const drafts = this.get();
+        return Object.keys(drafts)
+            .filter(
+                (draft_id) =>
+                    drafts[draft_id].type === "stream" &&
+                    drafts[draft_id].stream === stream &&
+                    drafts[draft_id].topic === topic,
+            )
+            .sort((draft_a, draft_b) => drafts[draft_a].updatedAt - drafts[draft_b].updatedAt);
+    };
+
+    exports.getDraftsIdByRecipients = function (private_message_recipient) {
```

Comment:
Hmm, this class uses camelCase naming, so I guess adding this isn't a regression.